scripts/install_cvmfs_2.10.1_eessi_AL2023.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Maybe add a header: "this file is part of ..., its purpose is to install CernVM-FS on Amazon Linux 2023 ... author: Alan O'Cais (@ocaisa) ... license"

Reply:
Yeah, it's not obvious that this is for Amazon Linux, I would have no idea what AL stands for.

Maybe script should also be renamed to `install_cvmfs_2.10.1_eessi_AmazonLinux-2023.sh`?

Reply:
I actually had that and switched as that is the shorthand that they use, I can switch back in the next PR